Dockerfile

```diff
@@ -11,9 +11,15 @@ RUN dnf install -y \
         rubygem-asciidoctor && \
     dnf groupinstall -y development-tools && \
     gem install \
+        asciidoctor-tabs:1.0.0.beta.6 \
+        asciidoctor:2.0.23 \
```

Comment:
If you do this, then you don't need `rubygem-asciidoctor` anymore.

Reply:
I have tested this but it failed locally. Removing this Gem works though -> pushed.